modules/writing_observer/writing_observer/aggregator.py

```diff
@@ -285,6 +285,7 @@ async def fetch_doc_from_google(student, doc_id):
         if student is None or doc_id is None or len(doc_id) == 0:
             return None
         import learning_observer.google
+        from learning_observer.stream_analytics.fields import KeyField, KeyStateType, EventField
```

Comment:
Make sure these are imported above in the function.

Reply:
Done